content/articles/building-a-simple-translation-app-using-python-for-beginners/index.md

```diff
@@ -0,0 +1,151 @@
+Want to build your very own language translation app?
```

Comment:
Please add the front matter to this article. You can find one [here](https://raw.githubusercontent.com/section-engineering-education/engineering-education/master/content/articles/boosting-algorithms-python/index.md) and edit yours according to that.